NSS-Day-Cohort-49/react-nutshell-state-slingers
# Hcr cleanup

#### Changes Made
1. Modified file `articles.css` to include additional styling
1. Modified file `articlesform` to include cancel button and functionality.
​1. Modified file `registration` to include a username form text field and saving the username to the database

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout hcr-cleanup
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
